server/views/components/work-media/work-media.njk

```diff
@@ -12,5 +12,5 @@
     {% componentV2 'scroll-to-info', {elementId: 'work-info'}, {'work': true}, {extraClasses: ['flush-container-right','js-work-media-control', 'pointer-events-all'], extraIconClasses: ['icon--halfx', 'icon--white']} %}
   </div>
 
-  {% componentV2 'image-iiif', model.iiifModel, {test: true}, model.iiifData %}
+  {% componentV2 'image', model.iiifModel, {test: true}, model.iiifData %}
```

Comment:
is `{test: true}` supposed to be in here? Looks like something I put in, but doesn't sound like it was meant to make it into prod!

Reply:
don't know, it was already there, I can take it out